From e25b97ceb29aec064af103ded662838809067211 Mon Sep 17 00:00:00 2001
From: Zhao Qiang <qiang.zhao@nxp.com>
Date: Thu, 7 Apr 2016 09:27:27 +0800
Subject: [PATCH 026/146] config: ifc: add ifc support to ls1043

add CONFIG_MTD_CFI_BE_BYTE_SWAP to arm64 freescale.config
for ifc on ls1043 being big endian.

Signed-off-by: Zhao Qiang <qiang.zhao@nxp.com>
---
 arch/arm64/configs/freescale.config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/freescale.config b/arch/arm64/configs/freescale.config
index 6e667c8..5a87778 100644
--- a/arch/arm64/configs/freescale.config
+++ b/arch/arm64/configs/freescale.config
@@ -33,6 +33,7 @@ CONFIG_MTD_CFI_ADV_OPTIONS=y
 CONFIG_MTD_CFI_INTELEXT=y
 CONFIG_MTD_CFI_AMDSTD=y
 CONFIG_MTD_CFI_STAA=y
+CONFIG_MTD_CFI_BE_BYTE_SWAP=y
 CONFIG_MTD_PHYSMAP_OF=y
 CONFIG_MTD_NAND=y
 CONFIG_MTD_NAND_FSL_IFC=y
-- 
2.1.0.27.g96db324

